src/components/AcceleratorReports/MetricBox.tsx

```diff
@@ -305,51 +305,75 @@ const MetricBox = ({
+                        style={{
+                          marginLeft: '-1px',
+                          marginTop: '-1px',
+                        }}
+                      />
+                    )}
+                    {!!editing && (
+                      <Button
+                        icon='iconEdit'
+                        onClick={() => setProgressModalOpened(true)}
+                        priority='ghost'
+                        size='small'
+                        type='passive'
+                        style={{
+                          marginLeft: '-1px',
+                          marginTop: '-1px',
+                        }}
+                      />
+                    )}
                   </Box>
-                )}
-                {!!editing && isReportSystemMetric(metric) && metric.overrideValue && (
-                  <Button
-                    icon='iconUndo'
-                    onClick={() => setResetMetricModalOpened(true)}
-                    priority='ghost'
-                    size='small'
-                    type='passive'
-                    style={{
-                      marginLeft: '-1px',
-                      marginTop: '-1px',
-                    }}
-                  />
-                )}
-                {!!editing && (
-                  <Button
-                    icon='iconEdit'
-                    onClick={() => setProgressModalOpened(true)}
-                    priority='ghost'
-                    size='small'
-                    type='passive'
-                    style={{
-                      marginLeft: '-1px',
-                      marginTop: '-1px',
-                    }}
-                  />
-                )}
-              </Box>
-              {isReportSystemMetric(metric) && metric.overrideValue && (
-                <Typography fontSize={'14px'} color={theme.palette.TwClrTxtSecondary} paddingTop={1.5}>
-                  {strings.formatString(strings.OVERWRITTEN_ORIGINAL_VALUE, metric.systemValue)}
-                </Typography>
+
+                  {metric.overrideValue && (
+                    <Typography fontSize={'14px'} color={theme.palette.TwClrTxtSecondary} paddingTop={1.5}>
+                      {strings.formatString(strings.OVERWRITTEN_ORIGINAL_VALUE, metric.systemValue)}
+                    </Typography>
+                  )}
+                </>
+              ) : (
+                isStandardOrProjectMetric(record) && (
+                  <Box display={'flex'} alignItems={'center'}>
+                    <TextField
+                      type='text'
+                      label={strings.PROGRESS}
+                      value={record.value}
+                      id={'value'}
+                      onChange={(value: any) => onChange('value', value)}
+                      display={!editing}
+                      required={true}
+                    />
+                    {
+                      <Typography paddingTop={3} paddingLeft={0.5}>
```

Comment:
Not super important, but are the curly braces necessary here?

Reply:
No, I will remove them!